File: Ruby/fibonacci.rb
#Fibonacci program
def fibonacci(n)
    a = 0
    b = 1

    # Compute Fibonacci number in the desired position.
    n.times do
        temp = a
        a = b
        # Add up previous two numbers in sequence.
        b = temp + b
    end

    return a
end

# Write Fibonacci numbers in sequence as per the userinput.
puts "Enter the number:"
userinput = gets.chomp.to_i
result = fibonacci(userinput)
puts result

File: JavaScript/fibonacci.js
#Fibonacci program 
/**
 * @param {number} N
 * @return {number}
 */

var fib = function(N) {
  f = [0,1];                   //Storing the known values in array
  for (var i = 2; i <= N; i++) {
      f[i] = f[i -1] + f[i - 2]; //Logic to find the fibonacci of the given number 
  }
    return f[N];
};

fib(9);
